UNUMColorPicker/Classes/UNUMColorPickerViewController.swift

```diff
@@ -13,72 +13,82 @@ public protocol UNUMColorPickerDelegate: class {
     func cancel()
 }
 
-public class UNUMColorPickerView: UIView {
+public class UNUMColorPickerViewController: UIViewController {
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        setupCollectionView()
```

Comment:
`setupCollectionView` was being done in `setup()`, which is brittle since `setup` was a public function that could be called at any time. Really it needs to be done at this point so that nothing crashes. Suppose the user hadn't called `setup` at the right time (problem A), or even calls it twice for some reason (problem B). Both of these are reasons why proper injection is safer than a setup function -- because then it is impossible not to call it (solving problem A) and it can only be done once (solving problem B).